schema.sql

```diff
@@ -7,5 +7,6 @@ CREATE TABLE animals (
     escape_attempts INT NOT NULL,
     neutered        BOOLEAN NOT NULL,
     weight_kg       DECIMAL,
+    species         VARCHAR(100),
     PRIMARY KEY (id)
```

Comment:
- [ ] (Optional) You can add a column to a table by running the following command instead of manually adding the name to the table:

`ALTER TABLE <Table Name> ADD <column name> <column type>;`